### ISSUE TITLE
Sno options that accept longer data streams should use @file pattern

### ISSUE THREAD

**Issue by olsen232** [koordinates/sno#133]
Currently there is no pattern with how large chunks of data are supplied at the command line.
```
sno commit --message="MESSAGE"
sno commit --message-file=message.txt
sno resolve --with-file=resolution.geojson
```

The pattern should be:
```
sno commit --message=-  # read from stdin
sno commit --message="MESSAGE"  # read from command line
sno commit --message=@message.txt  # read from file
```
And similarly for `sno resolve` resolution files, etc.

**Comment by craigds** [comment]
Fixed via #138

**Comment by olsen232** [comment]
![](https://media2.giphy.com/media/kEKcOWl8RMLde/giphy.gif)
